v2/mongodb-to-googlecloud/docs/MongoDbToBigQuery/mongodb-to-bigquery-image-spec-metadata.json

```diff
@@ -37,8 +37,20 @@
       "is_optional": false,
       "paramType": "TEXT",
       "regexes": [
-        "^(FLATTEN|NONE)$"
+        "^(FLATTEN|NONE|UDF)$"
```

Comment:
Why does UDF need to be passed in as a userOption? The template can infer if a UDF is being used if the javascriptDocumentTransformGcsPath and javascriptDocumentTransformFunctionName parameters are null (or at least empty string).

What if a user does not want to FLATTEN but does want to use a UDF?

Reply:
Updated as suggested